js/form.js

```diff
@@ -80,11 +85,19 @@
   };
 
   formNode.addEventListener(`change`, onFormNodeChange);
+  formNode.addEventListener(`submit`, (evt) => {
+    window.data.upload(new FormData(formNode), window.reset.page);
+    window.reset.createMessageElement();
+    evt.preventDefault();
+  });
+
+  formResetButton.addEventListener(`click`, (evt) => {
+    window.reset.page();
+    evt.preventDefault();
+  });
```

Comment:
У этой кнопки и так **type=reset**, избыточно его превентить. От того, что форма сброситя дважды ничего криминального не произойдет, а **evt.preventDefault();** заставляет разработчика задуматься и осмотреться зачем отменяется событие по умолчанию
```suggestion
  formResetButton.addEventListener(`click`, window.reset.page);
```